lib/plugins.js

```diff
@@ -231,19 +231,19 @@ function plugins (settings, deviceId) {
   }
 
   function recordPing (serialNumber, deviceTime, rec) {
-    const r_mp = {
+    const rMachinePings = {
       id: uuid.v4(),
       device_id: deviceId,
       serial_number: serialNumber,
       device_time: deviceTime
     }
-    const r_d = {
+    const rDevices = {
       last_online: deviceTime
     }
 
     return Promise.all([
-      db.none(pgp.helpers.insert(r_mp, null, 'machine_pings')),
-      db.none(pgp.helpers.update(r_d, null, 'devices') + `WHERE device_id = '${deviceId}'`)
+      db.none(pgp.helpers.insert(rMachinePings, null, 'machine_pings')),
+      db.none(pgp.helpers.update(rDevices, null, 'devices') + `WHERE device_id = '${deviceId}'`)
```

Comment:
Missed this last time. You should let pgp do the interpolation of ``deviceId`` instead of using JS backticks. Otherwise the quotes won't be escaped properly in certain situations and it could even lead to sql injection attacks.

Also, I don't think we need the ``r`` prefix here. Why not just ``machinePings`` and ``devices``.